src/app/components/Common/Player/Player.tsx

```diff
@@ -139,6 +143,21 @@ class Player extends React.Component<PlayerProps> {
       <span className={cx(styles.albumName)}>{nowPlayingItem.attributes.albumName}</span>
     );
 
+    if (navigator.mediaSession && hasMeta) {
```

Comment:
This shouldn't be part of the render, it should be encapsulated in a `useEffect` depending on `[nowPlayingItem]`

Reply:
Never really used useEffect hooks before, understand the principle but not quite sure how to incorporate it here. May you provide me with some example on how it should be used here? Will gladly refactor it, seems like better way for sure